README.md

```diff
@@ -40,7 +40,7 @@ Compatibility is quite high:
 - Almost all SA-MP **scripting functions** and **callbacks** are
   implemented.
 - **Database** functions (`db_*`) are implemented.
-- SA-MP server **plugins** work unmodified.
+- SA-MP server **plugins** work unmodified, if they don't use memory hacking.
```

Comment:
Yeah this is very true, especially for plugins such as `Pawn.CMD`, which I had to modify for it to work in MTA by getting rid of the byte pattern lookup when compiling for such with a preprocessor macro 😅

Reply:
Tbh Pawn.CMD is a very light example of a plugin for which there are many non-memhacking replacements ([zcmd](https://github.com/YashasSamaga/I-ZCMD), [dc_cmd](https://sampforum.blast.hk/showthread.php?tid=432330) etc). Meanwhile there are really crucial memhack plugins which cannot be replaced and they kinda based today just to make any advanced functionality, like Pawn.RakNet. Sometimes I even think that similar tool would be useful for MTA as well (at least a module which would give ability to intercept incoming packets like bylletsync, unoccupied or other sync types serverside, giving ability to handle every single packet in your script).